week-2-wide-perspective/w02e03-pagination-problem/src/middlewares/pagination.middleware.js

```diff
@@ -0,0 +1,18 @@
+export function paginationMiddleware(skipName = 'skip', limitName = 'limit') {
+  return (req, res, next) => {
+    const { [skipName]: skip, [limitName]: limit } = req.query
```

Comment:
bombowo.
Dodatkowo - wisienka na torcie jakby się jeszcze dało wybrać miejsce z którego wyciągać paginacje
(ale to już dodatek)

że np. jedziemy z `body` a nie z `query` - bo tak :)

Reply:
Dodany ten szczegół 😎